src/main/java/com/tabnine/Initializer.java

```diff
@@ -29,6 +31,11 @@ public void beforeApplicationLoaded(@NotNull Application application, @NotNull S
 
     @Override
     public void appStarting(@Nullable Project projectFromCommandLine) {
+        StaticConfig.getSentryDns().ifPresent(sentryDns -> {
```

Comment:
What happens if I do not call Sentry.init ? Will calls to sentry error out or will they silently ignore?

Reply:
yes